autoagora/price_multiplier.py

```diff
@@ -104,6 +104,8 @@ async def price_bandit_loop(subgraph: str):
             loss = bandit.update_policy()
             if loss is not None:
                 logging.debug("Price bandit %s - Training loss: %s", subgraph, loss)
+    except asyncio.CancelledError:
+        logging.debug("Price bandit %s - Removing bandit loop", subgraph)
```

Comment:
I checked [the official docs](https://docs.python.org/3/library/asyncio-task.html#task-cancellation), and it says *not* to suppress `CancelledError`, and to re-raise it if it's been caught. Thanks for making me look more carefully at that piece of code!

```suggestion
    except asyncio.CancelledError as cancelledError:
        logging.debug("Price bandit %s - Removing bandit loop", subgraph)
        raise cancelledError
```

Reply:
ok, added the raise as instruction. no unwanted effect